operator/config/rbac/role.yaml

```diff
@@ -5,6 +5,14 @@ metadata:
   creationTimestamp: null
   name: manager-role
 rules:
+- apiGroups:
```

Comment:
deployments, replicasets and statefulsets could be put into one block here like this:
```
- apiGroups:
  - apps
  resources:
  - deployments
  - replicasets
  - statefulsets
  verbs:
  - get
  - list
  - watch
```

Reply:
I know, but this is auto-generated by kubebuilder ...